tests/unit/unicorn_test.h

```diff
@@ -4,6 +4,7 @@
 #include <stdio.h>
 #include <stdint.h>
 #include <unicorn/unicorn.h>
+#include "qemu/bswap.h"
```

Comment:
This is wrong since we don't import qemu things in test files as they are not linked against the test targets, though it may work...